### PR TITLE
purifying longswords - alt title "there's no way this should have worked"

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
allows purifying eldritch longswords like you could bloody bastard swords
(it makes a purified longsword which has 25 force and other claymore stats)
## Why It's Good For The Game
allows the new strat of dabbing on cultists by purifying their swords and then using those to beat the shit out of them
## Changelog
:cl:
balance: You can now purify eldritch longswords with a bible. This creates purified longswords, which do not have anti-magic properties, but are still good for swinging at cultists.
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/book.dm [hunk 1]
```diff
@@ -173,22 +173,24 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "bible",
 			var/unholy2clean = A.reagents.get_reagent_amount(/datum/reagent/fuel/unholywater)
 			A.reagents.del_reagent(/datum/reagent/fuel/unholywater)
 			A.reagents.add_reagent(/datum/reagent/water/holywater,unholy2clean)
-	if(istype(A, /obj/item/twohanded/required/cult_bastard) && !iscultist(user))
-		var/obj/item/twohanded/required/cult_bastard/sword = A
+	if(istype(A, /obj/item/twohanded/required/cult_bastard) || istype(A, /obj/item/melee/cultblade) && !iscultist(user))
+		var/isbastardsword = FALSE
+		var/obj/item/sword = A
```
Comment (Ghommie) [inline code/game/objects/items/storage/book.dm]: These two lines are fairly useless now.

File: code/game/objects/items/storage/book.dm [hunk 2]
```diff
@@ -173,22 +173,24 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "bible",
...
 			playsound(src,'sound/effects/pray_chaplain.ogg',60,1)
-			for(var/obj/item/soulstone/SS in sword.contents)
-				SS.usability = TRUE
-				for(var/mob/living/simple_animal/shade/EX in SS)
-					SSticker.mode.remove_cultist(EX.mind, 1, 0)
-					EX.icon_state = "ghost1"
-					EX.name = "Purified [EX.name]"
-				SS.release_shades(user)
-				qdel(SS)
+			if(isbastardsword)
```
Comment (Ghommie) [inline code/game/objects/items/storage/book.dm]:
```suggestion
			if(istype(A, /obj/item/twohanded/required/cult_bastard))
```

**Review by Ghommie — CHANGES_REQUESTED** [review]

**Comment by necromanceranne** [comment]
I mentioned this to Hatterhat already but making it so that you can potentially start seeding among the crew even more nullrods is not a good move. We have claymores that aren't carrying the antimagic abilities that would be exclusive to the chaplain's nullrod, and even if they are significantly more potent than eldritch longswords in force (40 over the longsword's 30, has block compared to no block), they're being used against opponents who are very, very good at melee combat and generally have the advantage still. The loss of just one of the swords that the cult produces infinitely via a spell just ends up being disastrous.

I'd even recommend just making a new item that is a 30 force sword. Once it's merged on tg, there is a pr I have there that updates the claymore sprites that probably would be worth porting for that.

**Comment by Ghommie** [comment]
Oh, yea, more or less what the person above said. Null rod rewards for dunking cultists will only results in the cult being snowballed harder. Perhaps 20 damage or so not so holy claymores would be better.

File: code/game/objects/items/storage/book.dm [hunk 3]
```diff
@@ -173,23 +173,26 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "bible",
...
+					for(var/mob/living/simple_animal/shade/EX in SS)
+						SSticker.mode.remove_cultist(EX.mind, 1, 0)
+						EX.icon_state = "ghost1"
+						EX.name = "Purified [EX.name]"
+					SS.release_shades(user)
+					qdel(SS)
+				new /obj/item/nullrod/claymore(get_turf(A))
+				user.visible_message("<span class='notice'>[user] has purified [A]!</span>")
+				qdel(A)
+			new /obj/item/claymore/purified(get_turf(A))
```
Comment (Ghommie) [inline code/game/objects/items/storage/book.dm]: Replace these three lines with:
```
		else
			new /obj/item/claymore/purified(get_turf(A))
```

**Review by Ghommie — CHANGES_REQUESTED** [review]

**Review by Ghommie — APPROVED** [review]

**Pull request merged**